index.js

```diff
@@ -67,6 +67,84 @@ function toggleColor(element) {
     }
 }
 
+header.addEventListener('click', function(){
+ if(header.style.color === 'blue'){
+    header.style.color = 'black'
+    }  
+ else {
+     header.style.color = 'blue'
+    }  
+})
+
 /***** Deliverable 2 *****/
 
+const buttons = document.querySelector('button.like-button')
+buttons.addEventListener('click', function(){
+    //console.log('clicked!!!')
+    //const likes = document.querySelector('p.likes')
+    //const likes = button.previousElementSibling
+    // const currLikes = parseInt(likes.textContent)
+    // likes.textContent = `${currLikes + 1} Likes`
+    likes.textContent = `${traveler.likes++} Likes`
+})
+
+//header.addEventListener('click', function(event){
+//toggleColor(event.target)
+//})
 /***** Deliverable 3 *****/
+
+//const form = document.querySelector('new-animal-sighting-form')
+
+// form.addEventListener('submit', function (event) {
+
+// event.preventDefault()
+// console.log('Form submitted!')
+
+// console.dir(event.target)
+
+// const inputFieldOne = event.target[0]
+// const inputFieldTwo = event.target[1]
+// const inputFieldThree = event.target[2]
+// const inputFieldFour = event.target[3]
+
+// const newAnimalSighting = {
+//     Species: inputFieldOne,
+//     Photo: inputFieldTwo,
+//     link: inputFieldThree,
+//     Description: inputFieldFour,
+//     id: animalsArray[animalsArray.length - 1].id + 1
+//     }
+
+//     animalsSighting.push(newAnimalsObject)
+
+//     renderOnePost(newAnimalsObject)
+//     form.reset()
+// })
```

Comment:
This is great! Good work! It's a good idea you had to include this line: 

```js
animalsSighting.push(newAnimalsObject)
```

This is important in this problem because of the way you're creating custom IDs.